### PR TITLE
Release 22.1

### PR DESCRIPTION
**Pull request by aradi**

### CODE REVIEW

File: INSTALL.rst [hunk 1]
```diff
@@ -111,7 +138,7 @@ libraries.
 
 Notes:
 
-[1] Only Debug build is tested regulary with OpenMP turned off.
+[1] Only Debug build is tested regulary with OpenMP turned off and without ELSI.
```
Comment (bhourahine) [inline INSTALL.rst:141]:
```suggestion
[1] Only Debug build is tested regularly with OpenMP turned off and without ELSI.
```

File: README.rst [hunk 2]
```diff
@@ -21,19 +21,35 @@ Installation
 Obtaining via Conda
 -------------------
 
-The preferred way of obtaining DFTB+ is to install it via the conda package
+The preferred way of obtaining DFTB+ is to install it via the Conda package
```
Comment (bhourahine) [inline README.rst:24]: Their organization uses lower case for conda. https://docs.conda.io/projects/conda/en/latest/user-guide/index.html
```suggestion
The preferred way of obtaining DFTB+ is to install it via the conda package
```

File: README.rst [hunk 3]
```diff
@@ -21,19 +21,35 @@ Installation
...
 management framework using `Miniconda
 <https://docs.conda.io/en/latest/miniconda.html>`_ or `Anaconda
 <https://www.anaconda.com/products/individual>`_. Make sure to add/enable the
 ``conda-forge`` channel in order to be able to access DFTB+. (Please consult the
-conda documentation for how to set-up your conda environment.)
+Conda documentation for how to set-up your Conda environment.)
```
Comment (bhourahine) [inline README.rst:29]: as above.
```suggestion
conda documentation for how to set-up your conda environment.)
```

File: README.rst [hunk 4]
```diff
@@ -21,19 +21,35 @@ Installation
...
 
-We provide several build variants (mpi, non-mpi), choose the one suiting your
-needs. For example, by issuing ::
+We recommend to use the `Mamba installer <https://mamba.readthedocs.io/>`_, as
```
Comment (bhourahine) [inline README.rst:31]:
```suggestion
We recommend the use of the `Mamba installer <https://mamba.readthedocs.io/>`_, as
```

File: README.rst [hunk 5]
```diff
@@ -21,19 +21,35 @@ Installation
...
+
+or ::
+
+  mamba install 'dftbplus=*=mpi_mpich_*'
+
+or ::
+
+  mamba install 'dftbplus=*=mpi_openmpi_*'
+
+to get the last stable release of DFTB+ with serial(OpenMP-threaded) build or
```
Comment (bhourahine) [inline README.rst:50]:
```suggestion
to get the last stable release of DFTB+ with, respectively, serial (OpenMP-threaded) build or
```

File: README.rst [hunk 6]
```diff
@@ -21,19 +21,35 @@ Installation
...
+with MPI-parallelized build using the MPICH or the Open MPI framework,
```
Comment (bhourahine) [inline README.rst:51]:
```suggestion
with MPI-parallelized build using either the MPICH or the Open MPI framework,
```

**Review by awvwgk — APPROVED** [review]

**Pull request merged**